src/lazy/any_encoding.rs

```diff
@@ -724,6 +751,24 @@ impl<'top> LazyRawValue<'top, AnyEncoding> for LazyRawAnyValue<'top> {
             Binary_1_1(v) => Ok(v.read()?.into()),
         }
     }
+
+    fn annotations_span(&self) -> Span<'top> {
+        match &self.encoding {
+            LazyRawValueKind::Text_1_0(v) => v.annotations_span(),
+            LazyRawValueKind::Binary_1_0(v) => v.annotations_span(),
+            LazyRawValueKind::Text_1_1(v) => v.annotations_span(),
+            LazyRawValueKind::Binary_1_1(v) => v.annotations_span(),
+        }
+    }
+
+    fn value_span(&self) -> Span<'top> {
+        match &self.encoding {
+            LazyRawValueKind::Text_1_0(v) => v.value_span(),
+            LazyRawValueKind::Binary_1_0(v) => v.value_span(),
+            LazyRawValueKind::Text_1_1(v) => v.value_span(),
+            LazyRawValueKind::Binary_1_1(v) => v.value_span(),
+        }
+    }
```

Comment:
🗺️ The `span()` method returns a byte slice containing the complete value--both annotations _and_ data.

These methods allow you to access the annotations and data separately.